lib/rom/repository/command_proxy.rb

```diff
@@ -11,11 +11,10 @@ class CommandProxy
 
       def initialize(command)
         @command = command
-        @root = Dry::Core::Inflector.singularize(command.name.relation).to_sym
       end
 
       def call(input)
-        command.call(root => input)
+        command.call(command.name.relation => input)
       end
```

Comment:
> So I'd say it's better to use input evaluator when there's actually something to evaluate, otherwise just pass the input as it is.

Completely agree. Lets do that.

> tweak these two lines so that if `path` is `false` then we use an input evaluator that will simply return the input (literally -> input { input })

Why not when path is empty?